From f6d3a23f8741cbd883b1948b988a7d491c0da7f9 Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Tue, 21 Jan 2020 20:55:23 +0100
Subject: Split Automake for DejaGnu smoke config

Signed-off-by: Stephan Linz <linz@li-pro.net>

diff --git a/configure.ac b/configure.ac
index db89430..c1f477a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -289,6 +289,7 @@ AC_CONFIG_FILES([
 	data/Makefile
 	man/Makefile
 	src/Makefile
+	src/smoke/Makefile
 ])
 AC_OUTPUT
 
diff --git a/src/Makefile.am b/src/Makefile.am
index bdd259a..a44b5ea 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -24,24 +24,9 @@ AM_CPPFLAGS = -I$(top_srcdir)/src
 EXTRA_DIST =
 CLEANFILES =
 
-AUTOMAKE_OPTIONS = dejagnu
+SUBDIRS = . smoke
 
-EXTRA_DIST +=				\
-	smoke/mkcg/6416.exp		\
-	smoke/data/6416_30.xpm		\
-	smoke/mkcg/vid2k.exp		\
-	smoke/data/vid2k_30.xpm		\
-	smoke/config/default.exp
-
-DEJATOOL = mkcg
-RUNTESTDEFAULTFLAGS =			\
-	--tool "$$tool"			\
-	--srcdir "$$srcdir/smoke"	\
-	--status --all --verbose
-
-check_PROGRAMS =			\
-	mkcg.6416			\
-	mkcg.vid2k
+check_PROGRAMS = mkcg.6416 mkcg.vid2k
 
 ## TODO:9000 provide libmkcg as public library after refactoring
 noinst_LTLIBRARIES = libmkcg.la
diff --git a/src/smoke/Makefile.am b/src/smoke/Makefile.am
new file mode 100644
index 0000000..9a52fda
--- /dev/null
+++ b/src/smoke/Makefile.am
@@ -0,0 +1,40 @@
+#
+# Automake definitions for src/smoke/
+#
+# Copyright (C) 2002-2015  Stephan Linz <linz@li-pro.net>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA  02110-1301, USA.
+#
+
+EXTRA_DIST =
+CLEANFILES =
+
+AUTOMAKE_OPTIONS = dejagnu
+
+EXTRA_DIST +=							\
+	mkcg/6416.exp						\
+	data/6416_30.xpm					\
+	mkcg/vid2k.exp						\
+	data/vid2k_30.xpm					\
+	config/default.exp
+
+DEJATOOL = mkcg
+RUNTESTFLAGS =							\
+	--srcdir "$(abs_srcdir)"				\
+	--objdir "$(abs_top_builddir)/src"			\
+	--status --all --verbose
+
+MAINTAINERCLEANFILES = $(srcdir)/Makefile.in
